docs/generators/python-aiohttp.md

```diff
@@ -42,23 +42,29 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 <li>Dict</li>
 <li>List</li>
 <li>bool</li>
-<li>byte</li>
-<li>bytearray</li>
+<li>bytes</li>
 <li>date</li>
 <li>datetime</li>
+<li>dict</li>
 <li>file</li>
 <li>float</li>
 <li>int</li>
+<li>list</li>
 <li>object</li>
 <li>str</li>
 </ul>
 
 ## RESERVED WORDS
 
 <ul class="column-ul">
+<li>all_params</li>
```

Comment:
Why is all_params a reserved word?

Reply:
Noticed that too. Will file another PR to clean up the reserved word list.